apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.IntegrationTests/DatasetsTest.cs

```diff
@@ -92,6 +92,26 @@ public void CreateDataset_InitialEtag()
             Assert.Equal(created.Resource.ETag, fetched.Resource.ETag);
         }
 
+        [Fact]
+        public void CreateDataset_WithCustomDefaultLocation()
+        {
+            var client = BigQueryClient.Create(_fixture.ProjectId).WithDefaultLocation(_fixture.CustomLocation);
+            var id = _fixture.CreateDatasetId();
+
+            var created = client.CreateDataset(id);
+            Assert.Equal(_fixture.CustomLocation, created.Resource.Location);
+        }
+
+        [Fact]
+        public void CreateDataset_WithCustomLocation()
+        {
+            var client = BigQueryClient.Create(_fixture.ProjectId);
+            var id = _fixture.CreateDatasetId();
+
+            var created = client.CreateDataset(id, new CreateDatasetOptions { Location = _fixture.CustomLocation } );
+            Assert.Equal(_fixture.CustomLocation, created.Resource.Location);
+        }
```

Comment:
It might be good to also have a test which give the client a default location, but then overrides it on the call, and makes sure the call's value is used (same with the tests below).

Reply:
Done. (By "tests above" I assume you mean QueryTest? Not sure. I've added it to QueryTest though.)

Reply:
Yes, sorry, "below" as in further down in the PR's "Files changed" tab when viewing all changes :)